Attendance/Projects/Presentation/Presentation/Sources/CoreMember/QrCode/View/QrCodeView.swift

```diff
@@ -13,151 +13,145 @@ import CoreImage.CIFilterBuiltins
+        store.send(.async(.fetchEvent))
+        store.send(.async(.observeEvent))
       }
     }
+    .onChange(of: store.eventModel) { oldValue , newValue in
+      store.send(.async(.updateEventModel(newValue)))
+    }
+    
+    .sheet(item: $store.scope(state: \.destination?.makeEvent, action: \.destination.makeEvent)) { makeEventStore in
+      MakeEventView(store: makeEventStore, completion: {
+        store.send(.view(.closeMakeEventModal))
+      })
+      .presentationDetents([.height(UIScreen.screenHeight * 0.65)])
+      .presentationCornerRadius(20)
+      .presentationDragIndicator(.hidden)
+    }
+  }
 }
 
 extension QrCodeView {
-    
-    @ViewBuilder
-    fileprivate func generateQrImage() -> some View {
-        VStack {
-            
-            Spacer()
-                .frame(height: UIScreen.screenHeight * 0.2)
-            
-            if ((store.eventID?.isEmpty) != nil) {
-              if let qrCodeImage = store.qrCodeImage {
-                qrCodeImage
-                  .interpolation(.none)
-                  .resizable()
-                  .scaledToFit()
-                  .clipShape(RoundedRectangle(cornerRadius: 8))
-                  .frame(width: 200, height: 200)
-                } else {
-                  Image(asset: .appLogo)
-                    .resizable()
-                    .scaledToFit()
-                    .frame(width: 200, height: 200)
-                }
-            } else {
-                RoundedRectangle(cornerRadius: 8)
-                    .fill(Color.gray800.opacity(0.4))
-                    .frame(width:  200, height: 200)
-                    .overlay {
-                        VStack {
-                            Spacer()
-                            Image(asset: .qrCode)
-                                .resizable()
-                                .scaledToFit()
-                                .frame(width: 96, height: 96)
-                            Spacer()
-                        }
-                        
-                    }
-            }
+  
+  @ViewBuilder
+  fileprivate func generateQrImage() -> some View {
+    VStack {
+      
+      Spacer()
+        .frame(height: UIScreen.screenHeight * 0.2)
+      
+      if ((store.eventID?.isEmpty) != nil) {
+        if let qrCodeImage = store.qrCodeImage {
+          qrCodeImage
+            .interpolation(.none)
+            .resizable()
+            .scaledToFit()
+            .clipShape(RoundedRectangle(cornerRadius: 8))
+            .frame(width: 200, height: 200)
+        } else {
+          Image(asset: .appLogo)
+            .resizable()
+            .scaledToFit()
+            .frame(width: 200, height: 200)
         }
-    }
-    
-    @ViewBuilder
-    fileprivate func qrCodeReaderText() -> some View {
-        if store.eventModel != [ ] {
+      } else {
+        RoundedRectangle(cornerRadius: 8)
+          .fill(Color.gray800.opacity(0.4))
+          .frame(width:  200, height: 200)
```

Comment:
```suggestion
          .frame(width: 200, height: 200)
```
whitespace가 한칸 더 들어갔어요!!